src/pages/dashboard/index.tsx

```diff
@@ -1,15 +1,151 @@
-import React, {useState} from "react";
-import { Grid } from "@material-ui/core";
-// import dashboard from "../../api/dashboard";
+import React, {useEffect, useState} from "react";
+import {Grid} from "@material-ui/core";
+import PanelList from "../../components/PanelList";
+import FilterInput from "../../components/FilterInput";
+import List from "../../components/List";
+import dashboard from "../../api/dashboard";
 
 export const DashboardPage = () => {
-  const [data] = useState(null);
+  /**
+   * Set our initial useStates
+   */
+  const [items, setItems] = useState([]);
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [availableItems, setAvailableItems] = useState([]);
+  const [myListItems, setMyListItems] = useState([]);
+  const [filterValue, setFilterValue] = useState("");
 
-  return data === null ? null : (
+  /**
```

Comment:
Really excellent jsdoc comments! Makes me want to go back and add some to my code 😄